Sakura/core/payments.py

```
@@ -0,0 +1,116 @@
+                        'chat_id': update.message.chat.id,
+                        'text': refund_msg,
+                        'message_effect_id': random.choice(EFFECTS),
+                        'parse_mode': 'HTML',
+                        'reply_markup': reply_markup.to_json()
+                    }
+                    async with aiohttp.ClientSession() as session:
+                        async with session.post(url, json=payload) as response:
+                            result = await response.json()
+                            if not result.get('ok'):
+                                await update.message.reply_text(refund_msg, reply_markup=reply_markup)
+                except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)